tests/testthat/test-expression_convert.R

```diff
@@ -0,0 +1,126 @@
+test_that("simple expr with iteration var replaced", {
+  expr_test <- "log(eCount[i]) <- b0"
+  output <- expression_convert(expr_test)
+  
+  expect_equal(output, rlang::expr(log(eCount) <- b0))
```

Comment:
We could use snapshot testing to simplify the tests.

Reply:
Snapshot tests have been implemented